ray-operator/config/samples/ray-cluster-label-selector.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  name: ray-label-cluster
+spec:
+  enableInTreeAutoscaling: true
```

Comment:
why enable autoscaling? Is the example related to autoscaler? If not, please remove the related configs.